tests/referenceData/RecommendedLocationsTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
```

Comment:
@xianqiliu should we add the KO file as well or not? Let's make a decision so we can be consistent :)

Reply:
yes, we should make a decision, next week I will schedule a meeting with the team to review this question together :) @tsolakoua  @minjikarin @jabrena. As for this PR, @minjikarin please add the KO test for making sure the whole team understands how to code the KO test for API.

Reply:
Yes, we need to sit down to review the testing process. 🤩